source/renderer/app/components/wallet/WalletSendForm.js

```diff
@@ -628,7 +635,9 @@ export default class WalletSendForm extends Component<Props, State> {
         ];
       },
     ]);
-    this.form.$(newAsset).focus();
+    if (!this.isAddressFromSameWallet()) {
```

Comment:
@aleksandardjordjeviciohk this should be reverted.

Reply:
Updated